src/mmw/apps/modeling/mapshed/calcs.py

```diff
@@ -401,3 +401,44 @@ def sediment_phosphorus(nt_count):
     sedp = float(ag_sedp + nag_sedp) / total
 
     return sedp * 1.6
+
+
+def groundwater_nitrogen_conc(gwn_dict):
+    """
+    Calculate GwN and GwP from a dictionary of NLCD land use keys
+    paired with the number of cells as values
+    """
```

Comment:
Consider adding the original line numbers from where you adapted this code. For example, see here: https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/apps/modeling/mapshed/calcs.py#L383

The format I've been following is `filename@version:start-end`. For use, the filename is always `Class1.vb` and the version is always `1.3.0`.